scuttlebutt/src/state.rs

```diff
@@ -82,6 +85,7 @@ impl NodeState {
 
     fn set_with_version(&mut self, key: String, value: String, version: Version) {
         assert!(version > self.max_version);
+        assert!(value.bytes().len() <= MAX_KV_VALUE_SIZE, "Value too large.");
```

Comment:
Is there an easy way to have a more user-friendly error message:
"Value for key `{}` is too large (actual: {}, maximum: {})`.